pipelines/visitQualityCore.yaml

```diff
@@ -4,6 +4,7 @@ tasks:
   analyzeSourceTableCore:
     class: lsst.analysis.tools.tasks.SourceTableVisitAnalysisTask
     config:
+      connections.inputName: sourceTable_visit
```

Comment:
Technically not necessary if the task default has `defaultTemplates={"inputName": "sourceTable_visit"}`